samples/LocalizationSample/Startup.cs

```diff
@@ -14,7 +14,12 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
+            // Keep the following two lined in the same order. Or else, the `ResourcesPath` will not be guaranteed to be the same for both.
             services.AddJsonLocalization(options => options.ResourcesPath = "Resources");
```

Comment:
No need for this change while we don't use it. I'm still believing that unit tests is great